strongbox-web-core/src/main/java/org/carlspring/strongbox/domain/DirectoryContent.java

```diff
@@ -0,0 +1,92 @@
+package org.carlspring.strongbox.domain;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class DirectoryContent 
```

Comment:
If you assume this to be a MVC model POJO then it shouldn't contain `File` attributes. Attributes should be just Strings and other simple types. Hope that you understand what I mean.